### ISSUE TITLE
test-update branch: commit should happen only if certificates have changed

### ISSUE THREAD

**Issue by dolmen** [certifi/gocertifi#29]
It is not helpful to have automated commits of regeneration where no certificates changes happened, but instead only timestamp changes.

So the [Github Actions workflow](https://github.com/certifi/gocertifi/blob/test-update/.github/workflows/update.yml#L24) should check that the diff has at least more than 2 lines that changed.

**Comment by mattrobenolt** [comment]
Yeah, my plan was to just keep a file that was the hash of the certificate content, and compare to that on each run. Similar idea, and a bit better of a test than checking number of changed lines.

I just haven't had any motivation to do it to be honest.

**Comment by tmm1** [comment]
Friendly ping
